satellites.js

```diff
@@ -74,6 +74,24 @@ module.exports = function (RED) {
 
 		this.on('input', function (msg) {
 			var satellites = [];
+        // override tle1 if passed suitable value
+            if  ('satid' in msg)  {
+                if ( (typeof msg.satid === 'string') && (msg.satid.length < 1024) ) {
+                    node.satid = msg.satid
+                }
+            }
+        // override tle1 if passed suitable value
+            if  ('tle1' in msg)  {
+                if ( (typeof msg.tle1 === 'string') && (msg.tle1.length < 1024) ) {
+                    node.tle1 = msg.tle1
+                }
+            }
+        // override tle1 if passed suitable value
+            if  ('tle2' in msg)  {
```

Comment:
and finally...

Reply:
Ok, I will try to remember that.

Reply:
Do I need to do anything else with this?